src/components/Button/button.stories.tsx

```diff
@@ -0,0 +1,229 @@
+/* eslint-disable import/no-anonymous-default-export */
+/* eslint-disable import/no-default-export */
+
+import {
+  ArrowForwardIcon,
+  ChevronDownIcon,
+  EmailIcon,
+  PhoneIcon,
+  SearchIcon
+} from '@chakra-ui/icons'
+import {
+  Button,
+  ButtonGroup,
+  Container,
+  HStack,
+  IconButton,
+  Spinner,
+  Stack,
+  Wrap
+} from '@chakra-ui/react'
+import { FaFacebook, FaTwitter } from 'react-icons/fa'
+
+export default {
+  title: 'Forms/Button',
+  decorators: [
+    (Story: any) => (
+      <Container mt='40px'>
+        <Story />
+      </Container>
+    )
+  ]
+}
+
+export const basic = () => (
+  <Wrap>
+    <Button colorScheme='gray'>Button</Button>
```

Comment:
nitpick - but can we map all these rather than hard code them?

Reply:
We can try some different stuff and make adjustments